sdk/storage/Azure.Storage.Blobs/BreakingChanges.txt

```diff
@@ -1,5 +1,8 @@
 Breaking Changes
 ================
+Next Release
+--------------------------
+- PublicAccessType is no longer nullable. There is a new Enum value of None that replaces use of null.
```

Comment:
Nit - maybe tweak the wording to something more like `Changing from Nullable<PublicAccessType> to just PublicAccessType in all relevant APIs.  Use PublicAccessType.None instead of null.`

Reply:
Updated